### PR TITLE
Added old URL parsing approach for discord

### PR DESCRIPTION
**Pull request by utf94**
Added old parsing approach when there will be no url params to parse.

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ const makePromise = () => {
 };
 
 function parseQuery(queryString) {
+  if(!queryString){
```
Comment (avaer) [inline utils.js]: Why do we need early return?

Comment (avaer) [reply utils.js]: If it's not a string coming in, it's a type error and we should fix the bug.

If this is a string then the below code should be fine and this code doesn't do anything, right?

**Review by avaer — CHANGES_REQUESTED** [review]
Thanks! Just needs code cleanup.

File: utils.js (hunk 1 above)
Comment (utf94) [reply utils.js]: Resolved it by adding the check in the route.

**Review by avaer — APPROVED** [review]

**Pull request merged**